### PR TITLE
Caption for Grumpy Cat Slide

### PR DESCRIPTION
**Pull request by mikeyamato**
this PR adds a caption for the grumpy cat slide 

@dlowkeen 
@ykeanu 
@vmov031 

👀

### CODE REVIEW

File: _posts/0000-01-04-mikeyamato.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![angry-cat](https://cloud.githubusercontent.com/assets/16547949/25400569/064d7bc6-29c1-11e7-90dc-fef44edd9f0a.jpg)
 {: .center}
 
-Caption here
+So what if I haven't showered for 5 days. What are you, the shower police?
```
Comment (abechin) [inline _posts/0000-01-04-mikeyamato.md:9]: Is 5 days long enough?

Comment (mikeyamato) [inline _posts/0000-01-04-mikeyamato.md:9]: Looks great!

**Review by RKTMN — APPROVED** [review]
A+

**Review by githubteacher — APPROVED** [review]
Looks good! 👍

**Pull request merged**